solanches/models.py

```diff
@@ -235,6 +251,29 @@ def to_dict(self):
         cardapio = vars(self).copy()
         return cardapio
 
+    @staticmethod
+    def add_categoria(cardapio_id, categoria):
+        query = {"_id": cardapio_id}
+        cardapio = Cardapio.get_by_id(cardapio_id)
+        new_categoria = cardapio.get("categorias")
+        new_categoria += categoria if type(categoria) is list else [categoria]
+        new_values = {"$set": {"categorias": new_categoria}}
+        DB.cardapio.update_one(query, new_values)
+
+    @staticmethod
+    def remove_categoria(cardapio_id, categoria):
+        query = {"_id": cardapio_id}
+        cardapio = Cardapio.get_by_id(cardapio_id)
+        new_categoria = cardapio.get("categorias")
+        new_categoria.remove(categoria) if categoria in new_categoria else new_categoria
+        new_values = {"$set": {"categorias": new_categoria}}
```

Comment:
Isso deve ficar associado à uma variável, se não o else não tem efeito
```suggestion
        categories = cardapio.get("categorias")
        new_categories = categories.remove(categoria) if categoria in categories else categories
        new_values = {"$set": {"categorias": new_categories}}
```